cosim/black-parrot-example/ps.cpp

```diff
@@ -313,26 +316,25 @@ extern "C" void cosim_main(char *argstr) {
   bsg_pr_info("ps.cpp: MTIME delta (=1/8 BP cycles):    %'16llu (%16llx)\n",
               mtime_delta, mtime_delta);
   bsg_pr_info("ps.cpp: IPC        :                     %'16f\n",
-              ((double)minstrret_delta) / ((double)(mtime_delta)) / 8.0);
-  bsg_pr_info("ps.cpp: minstret (instructions retired): %'16llu (%16llx)\n",
-              counter_data, counter_data);
+              ((double)minstret_delta) / ((double)(mcycle_delta)));
   unsigned long long diff_ns =
       1000LL * 1000LL * 1000LL *
           ((unsigned long long)(end.tv_sec - start.tv_sec)) +
       (end.tv_nsec - start.tv_nsec);
   bsg_pr_info("ps.cpp: wall clock time                : %'16llu (%16llx) ns\n",
               diff_ns, diff_ns);
   bsg_pr_info(
-      "ps.cpp: sim/emul speed                 : %'16.2f BP cycles per minute\n",
+      "ps.cpp: sim/emul speed                         : %'16.2f BP cycles per minute\n",
       mtime_delta * 8 /
           ((double)(diff_ns) / (60.0 * 1000.0 * 1000.0 * 1000.0)));
 
   bsg_pr_info("ps.cpp: BP DRAM USAGE MASK (each bit is 8 MB): "
               "%-8.8x%-8.8x%-8.8x%-8.8x\n",
-              zpl->axil_read(0x2C + GP0_ADDR_BASE),
-              zpl->axil_read(0x28 + GP0_ADDR_BASE),
               zpl->axil_read(0x24 + GP0_ADDR_BASE),
-              zpl->axil_read(0x20 + GP0_ADDR_BASE));
+              zpl->axil_read(0x20 + GP0_ADDR_BASE),
```

Comment:
These addresses depend on the axil bus width. If we're at 32b for all FPGAs I guess it's fine for now